miter-vert.glsl

```diff
@@ -35,10 +35,11 @@ bool isNaN( float val ){
 void main() {
 	vec2 aCoord = aCoord, bCoord = bCoord, prevCoord = prevCoord, nextCoord = nextCoord;
 
-	// adjust scale for horizontal bars
-	vec2 scale = max(scale, MIN_DIFF);
-	vec2 scaleRatio = scale * viewport.zw;
+  vec2 adjustedScale = scale;
+  if (abs(scale.x) < MIN_DIFF) adjustedScale.x = MIN_DIFF; // * sign(scale.x);
+  if (abs(scale.y) < MIN_DIFF) adjustedScale.y = MIN_DIFF; // * sign(scale.y);
```

Comment:
Good catch re: redefined `scale`. Does it want the original sign (commented out here) or not? Again, consider removing the `if`:
`adjustedScale.x = max(abs(scale.x), MIN_DIFF) * sign(scale.x)`